### DISCUSSION TITLE
Survey: Which sync method are you using?

### DISCUSSION THREAD

**Discussion by marcelklehr** [floccusaddon/floccus#1439] [Polls]
In the absence of telemetry, I'd like to find out how popular the different sync methods are among users.

It would be awesome if you could click once for each method that you use. If you feel like you would like to give additional feedback, criticism or explanations, feel free to drop a comment below :)

Thanks 💙

**Comment by i2** [comment]
Voted! I would really like to store my bookmarks on GitHub/GitLab though. Great job. Thanks!

**Reply by i2** [reply]
@marcelklehr Thank you very much for implementing this <3

**Comment by mvozzo** [comment]
Please fix Floccus for home users that don't have a SSL certificate and are instead using a self-signed certificate.
  
I am stuck on Floccus v4.19.1 as it is the LAST version that still works with the latest versions of Chrome, Brave, MS Edge browsers with my Nextcloud Self-Signed Cert installation.  

Please like this comment if you agree.

**Reply by sn2020aug** [reply]
Is there a reason you can't get Let's Encrypt for your Nextcloud?

**Reply by marcelklehr** [reply]
Please see this thread on why I don't have any control about this: https://github.com/floccusaddon/floccus/issues/1560

**Reply by basxto** [reply]
> Is there a reason you can't get Let's Encrypt for your Nextcloud?

If you have a nextcloud at home that can’t be reached from the outside, the only option is to verify your domain via DNS. I’m not sure if all regstrars support that and Let’s Encrypt isn’t really meant for intranet services.

**Comment by Sa6SFKPH8** [comment]
I was using Nextcloud bookmarks, but have immediately transitioned to git based on the new 5.1.0 functionality. That option is not available on your survey above. Thanks for the terrific work on floccus.

PS: there is currently a minor spelling error in the Readme.md that floccus creates: 

"This repository is used to **syncrhonize** bookmarks via [floccus](https://floccus.org/)."

**Comment by roughnecks** [comment]
No multiple choices? WebDAV, Nextcloud and GIT :) Just to be sure, you know.

**Reply by marcelklehr** [reply]
hah, as in "sync is not backup but three syncs are"

**Comment by mvozzo** [comment]
Thanks for the Let's Encrypt suggestion. However, I don't think it is a
solution because I do not want my Nextcloud server to be accessible via the
public internet.

What if I ONLY want my Nextcloud Server to be accesses from within my home
network
I.E. http://192.168.x.x
 will Let's Encrypt enable this use case?

This is why I'm using a self-signed certificate as I do not want to expose
my Nextcloud Server to the public internet.

I'm sure there are many other home computer hobbyist and students that only
want to run Nextcloud on their home/private network.

**Reply by basxto** [reply]
If you own a domain, you can also use DNS-01 challenge for Let’s Encrypt. That adds a `txt` entry to your DNS and later removes it again. You don’t need a working webserver on that subdomain.
I even wrote the 192.168.. address into my DNS due to caching issues with intranet domains when using VPN.

**Comment by sn2020aug** [comment]
How about adding an entry for LoFloccus? It seems like a good way to sync within one's internal network.

**Reply by marcelklehr** [reply]
Mh, LoFloccus is WebDAV ;)

**Comment by RufioHugo** [comment]
I would have liked to use WebDAV => different Browsers on computer (Linux&Win) - one AddOn - no choice of "How to sync?" - so no chance to use it at all. On Android I can take my choice - but it won't work with my instance. Hence I found absolutely no FAQ/HowTo on floccus with WebDAV ... that's it, sorry. %-|

**Reply by marcelklehr** [reply]
If you like you can open an issue so I can help troubleshoot :)
